### PR TITLE
Use common address methods from Address.js

### PR DESCRIPTION
**Pull request by danfalc**
- moving common ID's to Address.js in single methods
- calling these methods in previously used places

### CODE REVIEW

File: cypress/integration/appoint_officer/appoint_corporate_secretary.spec.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ describe('Appoint a Corporate Secretary', () => {
 
         cy.accessibilityCheck();
         companyOverview.selectAllForms().selectLinkWithText('Directors and secretaries')
-            .selectLinkWithText('Appointment of corporate secretary - AP04');
+            .selectLinkWithText('Appointment of corporate secretary- AP04');
```
Comment (rsuller) [inline cypress/integration/appoint_officer/appoint_corporate_secretary.spec.js:12]: This is something that should be fixed by dev. It is inconsistent with the other options.

File: cypress/integration/mortgage_charges/cease_or_release_property_from_charge.spec.js [hunk 2]
```diff
@@ -52,8 +52,7 @@ describe('Cease or release property from a charge - MR05', () => {
         //Submit form without entering any information to fire error messages
         ceaseOrReleasePropertyFromCharge.expandAll()
         cy.accessibilityCheck();
-        ceaseOrReleasePropertyFromCharge.clickEnterAddressManually()
-        .enterInvalidCharactersToFireErrors("`")
+        ceaseOrReleasePropertyFromCharge.enterAddressManually()
```
Comment (rsuller) [inline cypress/integration/mortgage_charges/cease_or_release_property_from_charge.spec.js:55]: It's not something we currently validate but we have terminated most lines using `;` which is something we should probably aim to be consistent with. In this project at least.

File: cypress/integration/appoint_officer/appoint_corporate_secretary.spec.js (hunk 1 above)
Comment (danfalc) [reply cypress/integration/appoint_officer/appoint_corporate_secretary.spec.js]: Agree, reverted it back

File: cypress/integration/mortgage_charges/cease_or_release_property_from_charge.spec.js (hunk 2 above)
Comment (danfalc) [reply cypress/integration/mortgage_charges/cease_or_release_property_from_charge.spec.js:55]: Changed

**Pull request merged**